backend/back/src/webhooks/webhooks.module.ts

```diff
@@ -0,0 +1,12 @@
+import { Module } from "@nestjs/common";
+import { WebhooksController } from "./webhooks.controller";
+import { ConfigModule } from "@nestjs/config";
+import { GithubWebhookService } from "./services/gihub-webhook.service";
+import { GrpcModule } from "../grpc/grpc.module";
+
+@Module({
+	imports: [ConfigModule.forRoot(), GrpcModule],
```

Comment:
The config module is global you don't have to import it and surely not with forRoot

Reply:
Fixed